mast/templates/register.html

```diff
@@ -14,13 +14,11 @@ <h1 class="h2 mb-3">
         <p>Compete, track and enjoy</p>
     </div>
 
-    {{ render_field(form.username, autofocus=true) }}
+    {{ render_text_field(form.email, autofocus=true) }}
```

Comment:
Please use your defined constraints in form.py also on the frontend. We want to provide the user pre-evaluation before he sends the request on the server. (Obviously, password equality must be validated via Javascript, but this might be omitted.)